code/src/layouts/best-sellers/BestSellers.tsx

```diff
@@ -14,8 +15,10 @@ const BestSellers = () => {
     size: 5
   });
 
-  //@TODO Create Skeleton for component loading
-  if (isLoading) return <AppTypography>Loading...</AppTypography>;
+  const skeletonCards = Array.from(
+    { length: products?.content?.length || 10 },
+    (_, index) => <ProductSkeleton key={index} />
+  );
```

Comment:
I suggest creating new utility for creating some amount of skeleton

Reply:
I think in global `utils` folder. Maybe we can make it more generic to repeat any component any number of times

Reply:
I created a separate util function that creates skeletons.